dev/integration_tests/keyboard_hot_restart/README.md

```diff
@@ -0,0 +1,5 @@
+# keyboard_hot_restart
+
+An app used to verify that the keyboard is not visible after a hot restart.
```

Comment:
bonus: maybe also check system context menu is dismissed?

Reply:
The presence of the SystemContextMenu widget is probably enough to indicate it's showing or not, if you can check for that. If not, maybe you could print something from the TextField's contextMenuBuilder.

Reply:
Ahhh I didn't realize I could check for the context menu on the framework side, thanks for the pointers!

Reply:
Justin and I talked offline. Unfortunately, the presence of the `SystemContextMenu` widget and `TextField`'s `contextMenuBuilder` are not enough to detect whether the native context menu is visible. We don't have a good way to verify this native context menu from this "outside" test.